src/app.ts

```diff
@@ -0,0 +1,7 @@
+import { INestApplication } from '@nestjs/common';
```

Comment:
What is this used for?

Reply:
It's a solution so that I can get any service from outside a provider scope.